### PR TITLE
misc: Improve README and build instructions to favor Flatpak

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
# Description
Related: https://github.com/bottlesdevs/Bottles/issues/2921

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- See branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,63 +39,39 @@
...
+2. Run `flatpak run -d --filesystem=$PWD --command=bash com.usebottles.bottles//master` from the root of the repository, followed by `./utils/install.sh`. This will build Bottles and install it under the `build/` directory.
+3. Run `./build/bin/bottles` to launch Bottles
+
+Due to GNOME Builder limitations, Builder cannot build Bottles from the time being; see [GNOME/gnome-builder#2061](https://gitlab.gnome.org/GNOME/gnome-builder/-/issues/2061). This is the best workaround we can provide.
 
 ## Code of Conduct
-This project follows the [GNOME Code of Conduct.](https://wiki.gnome.org/Foundation/CodeOfConduct) You are expected to follow it in all Bottles spaces, such as this repository, project's social media, messenger chats and forums. Bigotry and harassment will not be tolerated.
+This project follows the [GNOME Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct). You are expected to follow it in all Bottles spaces, such as this repository, project's social media, messenger chats and forums. Bigotry and harassment will not be tolerated.
```
Comment (orowith2os) [inline README.md:74]:
```suggestion
This project follows the [GNOME Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct). You are expected to follow it in all Bottles spaces, such as this repository, the project's social media, messenger chats and forums. Bigotry and harassment will not be tolerated.
```

File: README.md [hunk 2]
```diff
@@ -36,63 +39,39 @@
...
+
+### Meson
+
+Since Bottles is primarily and officially distributed as a Flatpak, we only provide instructions to directly build it inside a Flatpak environment:
+
+1. Download and install the latest build of Bottles: [bottles-x86_64.zip](https://nightly.link/bottlesdevs/Bottles/workflows/build_flatpak/main/bottles-x86_64.zip). Unzip it, and run `flatpak install bottles.flatpak` (use `--user` if necessary)
+2. Run `flatpak run -d --filesystem=$PWD --command=bash com.usebottles.bottles//master` from the root of the repository, followed by `./utils/install.sh`. This will build Bottles and install it under the `build/` directory.
+3. Run `./build/bin/bottles` to launch Bottles
+
+Due to GNOME Builder limitations, Builder cannot build Bottles from the time being; see [GNOME/gnome-builder#2061](https://gitlab.gnome.org/GNOME/gnome-builder/-/issues/2061). This is the best workaround we can provide.
```
Comment (orowith2os) [inline README.md:71]:
```suggestion
Due to GNOME Builder limitations, Builder cannot build Bottles for the time being; see [GNOME/gnome-builder#2061](https://gitlab.gnome.org/GNOME/gnome-builder/-/issues/2061) for more context. This is the best workaround we can provide.
```

**Review by orowith2os — APPROVED** [review]

**Review by orowith2os — APPROVED** [review]

**Comment by orowith2os** [comment]
Not sure how to make the checks run, but everything LGTM

**Comment by TheEvilSkeleton** [comment]
I cancelled all the checks to avoid wasting GitHub's resources for no reason

**Pull request merged**